grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -1266,6 +1266,17 @@ class OrderService {
         }
 
         if (statements) {
+            try {
+                // Check if table exists.
+                dataService.executeQuery("SELECT * FROM order_summary_mv LIMIT 1")
```

Comment:
Are there other exceptions that might get thrown here? I guess it doesn't matter since an exception signifies a problem. Just wondering.